src/expt_metrics.py

```diff
@@ -111,94 +90,52 @@ def __str__(self):
         return self.message
 
 
-def validate_method(method):
-    if method not in VALID_METHODS:
-        msg = f'Request type must be one of: {VALID_METHODS}, actually: {method}'
-        print(msg)
-        raise ValueError(msg)
-    
-    return method
-
-
-@dataclass
-class ExptMetric:
-    ''' region object storing region name and min/max latitude bounds '''
-    name: str
-    cycle_start: datetime
-    cycle_stop: datetime
-    owner_id: str
-    group_id: str
-    experiment_type: str
-    platform: str
-    wallclock_start: datetime
-    wallclock_end: datetime
-    description: dict
-    experiment_data: ExperimentData = field(init=False)
-
-    def __post_init__(self):
-        print(f'in post init name: {self.name}')
-        if self.cycle_start > self.cycle_stop:
-            msg = f'start time must be before end time - start: {self.cycle_start}, ' \
-                f'end: {self.cycle_stop}'
-            raise ValueError(msg)
-        if self.platform not in VALID_PLATFORMS:
-            msg = f'\'platform\' must be one of {VALID_PLATFORMS}, was ' \
-                f'\'{self.platform}\''
-            raise ValueError(msg)
+# @dataclass
```

Comment:
also, where did ExptMetric class go? Was it removed by accident or on purpose?

Reply:
so I commented it out to prove it wasn't used in the workflow when running tests because I couldn't find any reference to it in the rest of the code; I should have fully deleted it because it's not needed.

Reply:
Sounds good.